src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -518,33 +518,37 @@ public static int findRepeatedSubstring(byte[] bases) {
      * @return                       Number of repetitions (0 if testString is not a concatenation of n repeatUnit's
      */
     public static int findNumberOfRepetitions(byte[] repeatUnit, byte[] testString, boolean lookForward) {
```

Comment:
Method is deprecated in GATK3 with the message: `@deprecated Move to use TandemRepeatFinder in protected (move to public if needed).`. Should we create a ticket in GATK4 for this task (migrating to `TandemRepeatFinder` here)?